test/tests/functional/pbs_mom_dynamic_resource.py

```diff
@@ -282,6 +282,7 @@ def test_delete_res(self):
         delete a resource while jobs are running/queued.
         """
 
+        message = "qmgr obj=foo svr=default: Resource busy on job"
```

Comment:
please remove  "qmgr obj=foo svr=default:" from the error message

Reply:
I agree.  That will allow the test to pass in API mode.